src/headers/Hero.scss

```diff
@@ -1,13 +1,35 @@
 .hero {
-  @apply bg-primary-darker;
-  @apply py-10;
-  @apply px-5;
-  @apply text-white;
-  @apply text-center;
-  @apply bg-cover;
-
-  @screen md {
-    @apply py-20;
+  --background-color: var(--bloom-color-primary-darker);
```

Comment:
With these changes does it mean that this component is V2-ified? If so should we add the red flag next to the storybook? I know that usually entails adding extra docs and what not as well

Reply:
Updated with the flag, but will wait on the docs since we're going to zero height and Jesse has been stubbing those out in a new format